python/pipeline/pupil.py

```diff
@@ -540,4 +575,463 @@ def make(self, key):
 
         self.insert1(key)
         for ckey in tqdm(contours):
-            self.Ellipse().insert1(ckey, ignore_extra_fields=True)
\ No newline at end of file
+            self.Ellipse().insert1(ckey, ignore_extra_fields=True)
+
+# If config.yaml ever updated, make sure you store the file name differently so that it becomes unique
```

Comment:
I didn't really checked the code below but I trust you did :)

Reply:
OK. In my mind, things are at least logically structured...